app/src/main/java/com/github/houseorganizer/houseorganizer/HouseSelectionActivity.java

```diff
@@ -0,0 +1,34 @@
+package com.github.houseorganizer.houseorganizer;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+public class HouseSelectionActivity extends AppCompatActivity {
+
+    RecyclerView housesView;
+    final String[] houseNames = {"House 1", "House 2", "House 3", "House 4"};
```

Comment:
We might want to customize house names in the future. But as this is only a place holder I think this is fine.

Reply:
In the future, we could to ask the user to pick a nickname for each household. In fact, this is how I structured the dummy firestore database, for now.

Reply:
I agree, it will be done when the database is ready.